google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmDeploymentRunConfigurationEditor.java

```diff
@@ -58,13 +58,11 @@
   private TextFieldWithBrowseButton dockerFilePathField;
   private JButton generateAppYamlButton;
   private JButton generateDockerfileButton;
-  private AppEngineHelper appEngineHelper;
 
 
   public ManagedVmDeploymentRunConfigurationEditor(final Project project,
       final DeploymentSource source, final AppEngineHelper appEngineHelper) {
     this.project = project;
-    this.appEngineHelper = appEngineHelper;
```

Comment:
That constructor parameter is actually required since it is this 'final' parameter that anonymous classes below will reference.